QueryCiv3/BiqSections/Govt.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    public struct GOVTGOVT {
+        public int CanBribe;
+        public int BriberyModifier;
+        public int ResistanceModifier;
+    }
+
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct GOVT
+    {
+        public int Length;
+        public int DefaultType;
+        public int TransitionType;
+        public int RequiresMaintenance;
+        public int Toggle1; // ??? 0: Republic/Democracy, 1: Other
+        public int TilePenalty;
+        public int TradeBonus;
+
+        private fixed byte Text[352];
+        public string Name { get => Util.GetString(ref this, 28, 64); }
```

Comment:
I don't think it is possible, because fixed fields can't be passed as arguments and I don't think there's a way to automatically reference Text's offset in the overall struct automatically. But yeah, I agree it's not ideal, and it might be worth looking into at least lessening the magic and improving the readability of it. Probably low priority though.

Reply:
That's kind of what I expected.  I might poke around at it some time, but agree it's not a high priority, particularly as these structures are not going to change (or at least not unless we add PTW support, and we'll have other things to think about if we do that).